evap/results/tests/test_views.py

```diff
@@ -27,6 +27,36 @@ def setUpTestData(cls):
         cls.semester = mommy.make(Semester, id=1)
 
 
+class TestResultsViewContributionWarning(WebTest):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.semester = mommy.make(Semester, id=3)
+        mommy.make(UserProfile, username='contributor', groups=[Group.objects.get(name='Staff')], email="contributor@institution.example.com")
+        contributor = UserProfile.objects.get(username="contributor")
```

Comment:
merge these two lines to `contributor = mommy.make...`?

Reply:
done